Test/MonoGame.Tests.XNA.csproj

```diff
@@ -143,6 +141,8 @@
     <Compile Include="Framework\Graphics\TextureCubeTest.cs" />
     <Compile Include="Framework\Graphics\VertexBufferTest.cs" />
     <Compile Include="Framework\Graphics\ViewportTest.cs" />
+    <Compile Include="Framework\Input\KeyboardTest.cs" />
+    <Compile Include="Framework\Input\MouseTest.cs" />
```

Comment:
Jep, I left it out because none of the tests compiled for XNA (most things we use to verify tests are not public API), but with the old tests merged at least one will. I'll add this back in and #if around failing tests. Thanks for the review Tom!